space.go

```diff
@@ -138,6 +147,28 @@ func importSpaceV2(source map[string]interface{}) (*space, error) {
 	}, nil
 }
 
+func importSpaceV3(source map[string]interface{}) (*space, error) {
```

Comment:
We should use the pattern where the import version is passed in to a common import function (as is done elsewhere).

Reply:
indeed, done